camunda-process-instances.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+import { getAccessToken } from "../auth.js";
+
+// An action that deploys one or more resources (e.g. processes, decision models, or forms). This is an atomic call, i.e. either all resources are deployed or none of them are.
+async function deployResources() {
+  console.log("deploying resources");
+}
+
+// An action that creates and starts an instance of the specified process.
+async function createInstance() {
```

Comment:
@pepopowitz For the body, do we need _everything_ outlined on https://docs.camunda.io/docs/next/apis-tools/camunda-api-rest/specifications/create-process-instance/?